app/components/back-office-app-services/applications-migration-function/provider.tf

```diff
@@ -2,8 +2,9 @@ terraform {
   required_providers {
     azurerm = {
       source                = "hashicorp/azurerm"
-      version               = "3.107.0"
-      configuration_aliases = [azurerm]
+      version               = ">= 3.0, < 5.0"
+      configuration_aliases = [azurerm, azurerm.tooling]

```

Comment:
azurerm.tooling may not be needed here

Reply:
Looks like possibly not, nothing before seems to have been referencing it.